include/WCSimDetectorConstruction.hh

```diff
@@ -71,6 +71,7 @@ public:
   G4String GetDetectorName()      {return WCDetectorName;}
   G4double GetWaterTubeLength()   {return WCLength;}
   G4double GetWaterTubePosition() {return WCPosition;}
+  G4double GetWaterTubeRadius()   {return WCRadius;}
```

Comment:
It seems useful though. How else do I retrieve the detector radius from 
a WCSimDetectorConstruction object? I need it for the 
feature/Radioactivity branch in order to distribute radioactive isotopes 
in the water volume.

Federico

On 20/06/16 20:17, eosullivan wrote:
>
> In include/WCSimDetectorConstruction.hh 
> <https://github.com/WCSim/WCSim/pull/168#discussion_r67749426>:
>
> > @@ -71,6 +71,7 @@ public:
> >    G4String GetDetectorName()      {return WCDetectorName;}
> >    G4double GetWaterTubeLength()   {return WCLength;}
> >    G4double GetWaterTubePosition() {return WCPosition;}
> > +  G4double GetWaterTubeRadius()   {return WCRadius;}
>
> This GetWaterTubeRadius() is some relic code that isn't in WCSim anymore.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/WCSim/WCSim/pull/168/files/6c154164188cb77bb4590da21bfcd48133f88281#r67749426>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ANMXw0I9GIMTgSLAYNkv0BNOgpEDj-94ks5qNudSgaJpZM4IutXq>.
>

-- 

_____________________________________________________________
Federico Nova         office: R1 2.89
phone: 01235 44 6980
email:  federico.nova@stfc.ac.uk
_____________________________________________________________
Particle Physics Department            Rutherford Appleton Laboratory
Harwell, Didcot, Oxford, OX11 0QX, UK
_____________________________________________________________